mesonbuild/compilers/cuda.py

```diff
@@ -577,12 +577,8 @@ def sanity_check(self, work_dir: str, env: 'Environment') -> None:
 
         # Run sanity check (if possible)
         if self.is_cross:
-            if not env.has_exe_wrapper():
-                return
-            else:
-                cmdlist = env.exe_wrapper.get_command() + [binary_name]
-        else:
-            cmdlist = self.exelist + ['--run', '"' + binary_name + '"']
+            return
+        cmdlist = self.exelist + ['--run', '"' + binary_name + '"']
```

Comment:
nitpick: I know it was already there, but imho it would be more readable using:
`f'"{binary_name}"'